package.json

```diff
@@ -58,7 +58,7 @@
     "request": "^2.61.0",
     "request-promise-native": "^1.0.4",
     "require-dir": "^0.3.2",
-    "superagent": "^1.3.0",
+    "superagent": "3.7.0",
```

Comment:
What do we use this for? This version bump seems much more major

Reply:
It's just used to make HTTP requests, still works for our purposes (simple POST)